scripts/launch/demo.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -eo xtrace
+set +eo xtrace
```

Comment:
What does it mean?

Reply:
This fixes the error I was getting before , @volhovm helped with this comment on slack
> So there's ` gen_kademlia_topology` function in `common-functions.sh`
> IT generates topology/kademlia files, but before it does it `rm -v`s them.
> The reason why it fails for you is because `set -e` is turned on i guess.
> `rm -v` returns nonzero codes which terminates the script. Just `set +e`, call the function, do `set -e` back > later.

Reply:
Ok, this change is no longer necessary after #2062. Please revert it.

Reply:
Done